src/python/grpcio/grpc/_server.py

```diff
@@ -65,12 +65,23 @@ def _serialized_request(request_event):
   return request_event.batch_operations[0].received_message.bytes()
 
 
-def _code(state):
```

Comment:
High level question:
Do the changes to _server.py solve issues with the GA API, or are they solely to facilitate adaptation between the Beta and GA APIs?  If the latter is the case, is there any way we can move this logic to the adapter?